docs/user-guide/expressions/structs.md

```diff
@@ -0,0 +1,86 @@
+# The Struct datatype
+
+Polars `Sturct`s are an excellent gateway to operations that require multiple columns. For this section, let's start with a `DataFrame` that captures the average rating of a few movies across some states in the U.S.: 
```

Comment:
Can you also note that moving columns into `structs` is a free operation. This will not copy data.

Reply:
Done.